sys/shell/commands/sc_icmpv6_echo.c

```diff
@@ -38,7 +38,11 @@ static char ipv6_str[NG_IPV6_ADDR_MAX_STR_LEN];
 
 static void usage(char **argv)
 {
-    printf("%s [<n>] <ipv6 addr> [<payload_len>]\n", argv[0]);
+    printf("%s [<count>] <ipv6 addr> [<payload_len>] [<delay in ms>]\n", argv[0]);
+    puts("defaults:");
+    puts("    count = 3");
+    puts("    payload_len = 4");
+    puts("    delay = 1000");
```

Comment:
It's the default on most other platforms, too. (Keep in mind: for <=200 ms on Linux you even need root).